wcfsetup/install/lang/de.xml

```diff
@@ -1313,6 +1313,38 @@ Das Fehlerprotokoll enthält {$data[count]} neue Einträge. Die ersten drei, in
+		<item name="wcf.acp.option.image_imagick_filter"><![CDATA[ImageMagick-Filter]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.point"><![CDATA[Point]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.box"><![CDATA[Box]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.triangle"><![CDATA[Triangle]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.hermite"><![CDATA[Hermite]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.hanning"><![CDATA[Hanning]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.hamming"><![CDATA[Hamming]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.blackman"><![CDATA[Blackman]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.gaussian"><![CDATA[Gaussian]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.quadratic"><![CDATA[Quadratic]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.cubic"><![CDATA[Cubic]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.catrom"><![CDATA[Catrom]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.mitchell"><![CDATA[Mitchell]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.jinc"><![CDATA[Jinc]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.sinc"><![CDATA[Sinc]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.sincfast"><![CDATA[SincFast]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.kaiser"><![CDATA[Kaiser]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.welsh"><![CDATA[Welsh]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.parzen"><![CDATA[Parzen]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.bohman"><![CDATA[Bohman]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.bartlett"><![CDATA[Bartlett]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.lagrange"><![CDATA[Lagrange]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.lanczos"><![CDATA[Lanczos]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.lanczossharp"><![CDATA[LanczosSharp]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.lanczos2"><![CDATA[Lanczos2]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.lanczos2sharp"><![CDATA[Lanczos2Sharp]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.robidoux"><![CDATA[Robidoux]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.robidouxsharp"><![CDATA[RobidouxSharp]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.cosine"><![CDATA[Cosine]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.spline"><![CDATA[Spline]]></item>
+		<item name="wcf.acp.option.image_imagick_filter.spline"><![CDATA[LanczosRadius]]></item>
```

Comment:
typo, should be lanczosradius not spline

Reply:
Fixed, ty.